src/plugins/openmeteo/datasource.py

```
@@ -0,0 +1,333 @@
+from typing import List, Optional, Dict, Any, AsyncIterator
```

Ruff (0.12.2)

Import from `collections.abc` instead: `AsyncIterator`

Import from `collections.abc`

(UP035)